ops/terraform/services/pipeline/user-data.sh.tftpl

```diff
@@ -13,17 +13,17 @@ cd /beneficiary-fhir-data/ops/ansible/playbooks-ccs/
 # TODO: Consider injecting ansible variables with more modern ansible versions. BFD-1890.
 aws ssm get-parameters-by-path \
     --with-decryption \
-    --path "/bfd/${env}/pipeline/shared/" \
+    --path "/bfd/${env}/pipeline/nonsensitive/shared/" \
```

Comment:
Checked ssm for e.g. /bfd/prod/pipeline/nonsensitive/shared, seems it is not provisioned yet?
is there provision to be done together with this PR?

Reply:
@JFU-NAVA-PBC  It was but since we just had a deployment, they got wiped out